java/rocksjni/portal.h

```diff
@@ -9089,5 +9089,155 @@ class BlockBasedTableOptionsJni
   }
 };
 
+class WideColumnJni {
+ public:
+  static void convertWideColumnsDirect(
+      std::function<void(ROCKSDB_NAMESPACE::Slice&,
+                         ROCKSDB_NAMESPACE::WideColumns&)>
+          op,
+      JNIEnv* env, jobject jKey, jint jKeyOffset, jint jKeyLen,
+      jobjectArray jNames, jintArray jNamesOffset, jintArray jNamesLen,
+      jobjectArray jValues, jintArray jvaluesOffset, jintArray jValuesLen) {
+    auto _key = env->GetDirectBufferAddress(jKey);
```

Comment:
Can we avoid `_` as a variable name prefix please. Perhaps `jKeyAddress` would be better?

Also this occurs below too...